pkg/gateway/api/api_grpc.go

```diff
@@ -769,12 +773,23 @@ func (s *grpcGatewayService) RegisterEvents(
 				}
 				continue
 			}
-			metricsMessages = append(metricsMessages, event)
+			m := &eventproto.MetricsEvent{}
+			if err := ptypes.UnmarshalAny(req.Events[i].Event, m); err != nil {
```

Comment:
Can we write this code like this?
```suggestion
			if err := ptypes.UnmarshalAny(event.Event, m); err != nil {
```

Reply:
https://qiita.com/uchiko/items/1c611f0db618ce9dc0a9